tech.json

```diff
@@ -180,6 +180,11 @@
               "note": "Moving along a 2-tile-high passage while standing up by repeatedly spin-jumping and then pressing down. Comes with softlock risks without Morph Ball."
             }
           ]
+        },
+        {
+          "name": "canMidairWiggle",
+          "requires": [],
+          "note": "Turning around quickly so as to manipulate game physics.  To remove WallJump state or to move sharpley underwater."
```

Comment:
Not sure I follow this. Can you try to clarify it a bit more?
Is the underwater aspect the same as canMomentumConservingTurnaround?

Reply:
I still think this is a bit vague, especially "to manipulate game physics". 
Maybe mention that this can be used with screw attack to break blocks immediately after wall jumping - I think that will be the majority of the uses for this and it could make it a bit more clear as to what this tech is.